l10n_br_fiscal/views/document_line_view.xml

```diff
@@ -89,6 +89,7 @@
                         <field name="icmssn_reduction" force_save="1" attrs="{'readonly': [('icmssn_tax_id', '!=', False)], 'invisible': [('tax_framework', '=', '3')]}"/>
                         <field name="icms_base" force_save="1" attrs="{'readonly': [('icms_tax_id', '!=', False)], 'invisible': [('tax_framework', 'in', ('1', '2'))]}"/>
                         <field name="icms_percent" force_save="1" attrs="{'readonly': [('icms_tax_id', '!=', False)], 'invisible': [('tax_framework', 'in', ('1', '2'))]}"/>
+                        <field name="icms_base_type"/>
```

Comment:
Tem que  colocar o attrs para não aparecer o campo quando o emitente for do Simples Nacional

Reply:
![image](https://user-images.githubusercontent.com/1975978/76119600-8bb4b900-5fce-11ea-823e-e9c8a14cf08f.png)

Este campo tb deve aparecer no caso do simples.